install/common-pre-install/pre-installation-cpi.md

```diff
@@ -0,0 +1,745 @@
+        Images resolved
+        Kubernetes Boot Selection:
+                kernel: /var/www/ephemeral/data/k8s/0.3.51/5.3.18-150300.59.87-default-0.3.51.kernel
+                initrd: /var/www/ephemeral/data/k8s/0.3.51/initrd.img-0.3.51.xz
+                squash: /var/www/ephemeral/data/k8s/0.3.51/secure-kubernetes-0.3.51.squashfs
+        Storage Boot Selection:
+                kernel: /var/www/ephemeral/data/ceph/0.3.51/5.3.18-150300.59.87-default-0.3.51.kernel
+                initrd: /var/www/ephemeral/data/ceph/0.3.51/initrd.img-0.3.51.xz
+                squash: /var/www/ephemeral/data/ceph/0.3.51/secure-storage-ceph-0.3.51.squashfs
+        Attempting to set all known BMCs (from /etc/conman.conf) to DHCP mode
+        current BMC count: 8
+        Waiting on 8 to request DHCP ...
+        All [8] expected BMCs have requested DHCP.
+        /root/bin/set-sqfs-links.sh is creating boot directories for each NCN with a BMC that has a lease in /var/lib/misc/dnsmasq.leases
+                NOTE: Nodes without boot directories will still boot the non-destructive iPXE binary for bare-metal discovery usage.
+                Images will be stored on the NCN at /run/initramfs/live/1.3.0-rc.3/
+        /var/www is ready.
+   ```
+
+   Goto `/var/www` and create additional symlinks - 
+
+   ``` 
+    cd /var/www
+    cwd=$PWD; for dir in $(ls|grep ncn-); do cd $dir; ln -sf ../ephemeral/data/k8s/0.3.59/secure-kubernetes-*.squashfs rootfs ; cd $cwd; done   
+    cd  /srv/tftpboot/
+    ln -sf /var/www/ncn-* .
+   ```
+
+1. (`pit#`) Verify that the LiveCD is ready by running the preflight tests.
+
+   ```bash
+   csi pit validate --livecd-preflight
+   ```
+   Expected Output - 
+   ```
+        Running LiveCD preflight checks (may take a few minutes to complete)...
+        Writing full output to /opt/cray/tests/install/logs/print_goss_json_results/20220929_101501.528062-22314-Z7D4bWt9/out
+
+        Reading test results for node gamora-ncn-m001-pit (suites/livecd-preflight-tests.yaml)
+
+        Checking test results
+        Only errors will be printed to the screen
+
+        GRAND TOTAL: 162 passed, 0 failed
+
+        PASSED
+   ```
+
+   If any tests fail, they need to be investigated. After actions have been taken to rectify the tests
+   (for example, editing configuration or CSI inputs), then restart from the beginning of the
+   [Initialize the LiveCD](#35-initialize-the-livecd) procedure.
+
+1. Save the `prep` directory for re-use.
+
+   This needs to be copied off the system and either stored in a secure location or in a secured Git repository.
+   There are secrets in this directory that should not be accidentally exposed.
+
+## Next topic
+
+After completing this procedure, proceed to configure the management network switches. The generated paddle file can be used as input to the canu command to configure the switches.
+
+See [Configure management network switches](README.md#6-configure-management-network-switches).
```

Comment:
move this section before 1. (`pit#`) Setup links to the boot artifacts extracted from the CSM tarball.

Reply:
Addressed in https://github.com/abhishek-g-suresh/docs-csm/pull/5